app/bundles/CoreBundle/Templating/Twig/Extension/EmailExtension.php

```diff
@@ -0,0 +1,214 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CoreBundle\Templating\Twig\Extension;
+
+use Mautic\CoreBundle\Helper\ListParser\ArrayListParser;
+use Mautic\CoreBundle\Helper\ListParser\BarListParser;
+use Mautic\CoreBundle\Helper\ListParser\Exception\FormatNotSupportedException;
+use Mautic\CoreBundle\Helper\ListParser\JsonListParser;
+use Mautic\CoreBundle\Helper\ListParser\ListParserInterface;
+use Mautic\CoreBundle\Helper\ListParser\ValueListParser;
+use Mautic\CoreBundle\Templating\Helper\AssetsHelper;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+
+class EmailExtension extends AbstractExtension
+{
+
+    /**
+     * @var AssetsHelper
+     */
+    protected AssetsHelper $helper;
+
+    /**
+     * @param AssetsHelper $helper
+     */
+    public function __construct(AssetsHelper $helper)
+    {
+        $this->helper = $helper;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('instanceof', [$this, 'isInstanceof']),
+            new TwigFunction('parseBooleanList', [$this, 'parseBooleanList']),
+            new TwigFunction('parseListForChoices', [$this, 'parseListForChoices']),
```

Comment:
Wouldn't this do the same?
```suggestion
            new TwigFunction('parseListForChoices', [AbstractFormFieldHelper::class, 'parseListForChoices']),
```
The duplicated method bellow can be removed.